crypto/poseidon/poseidon.go

```diff
@@ -165,7 +167,7 @@ func HashFixed(inpBI []*big.Int) (*big.Int, error) {
 	inp := utils.BigIntArrayToElementArray(inpBI[:])
 
 	state := make([]*ff.Element, t)
-	state[0] = zero()
+	state[0] = ff.NewElement().SetBigInt(domain)
```

Comment:
No need to update `HashWithCap` (used in `codehash`), right?

Reply:
Yes. Only `HashFixed` (used by zktrie) need to be updated